pkg/instance/instance.go

```diff
@@ -423,20 +423,21 @@ type OptionalFuzzerArgs struct {
 }
 
 type FuzzerCmdArgs struct {
-	Fuzzer    string
-	Executor  string
-	Name      string
-	OS        string
-	Arch      string
-	FwdAddr   string
-	Sandbox   string
-	Procs     int
-	Verbosity int
-	Cover     bool
-	Debug     bool
-	Test      bool
-	Runtest   bool
-	Optional  *OptionalFuzzerArgs
+	Fuzzer     string
+	Executor   string
+	Name       string
+	OS         string
+	Arch       string
+	FwdAddr    string
+	Sandbox    string
+	SandboxArg int
```

Comment:
It looks like `SandboxArg` belongs to the `Optional` set of arguments here.

The problem is that syzbot needs to be able to compile and run old syz-fuzzer and syz-execprog processes. We cannot just start passing new arguments to them -- the old binaries will immediately exit with an error about an unknown flag.